config/locales/ar.yml

```diff
@@ -19,8 +19,8 @@ ar:
         user:
           attributes:
             password:
-              too_short: قصيرة جدًا (الحد أدنى 6 أحرف)
-              too_weak: ضعيفة جدا؛ استخدم مزيجا من الحروف والأرقام والرموز.
+              too_short: قصيرة جدًا (الحد أدنى 8 أحرف)
+              too_weak: ضعيفة. يجب أن تتكون كلمة السر من ثمانية أو أكثر من الأحرف والأرقام. كلمة السر يجب ألا تطابق البريد الإلكتروني.
```

Comment:
Since the user will see the `too_short` message if it's less than 8 letters.
The message in too_week should just focus on it having letters and digits.
"يجب ان تحتوي كلمة السر على أرقام وأحرف" for example.

Reply:
Done.